google-http-client-xml/src/test/java/com/google/api/client/xml/XmlEnumTest.java

```diff
@@ -82,10 +72,34 @@ public void testParse_anyType() throws Exception {
     assertEquals(XML, out.toString());
   }
 
-  public void testParse_enumElementType() throws Exception {
-    XmlEnumTest.AnyTypeEnumElementOnly xml = new XmlEnumTest.AnyTypeEnumElementOnly();
+  /**
+   * The purpose of this test is to parse an XML element to an objects's member variable
+   */
+  @Test
+  public void testParseToEnumElementType() throws Exception {
+    assertEquals(XML_ENUM_ELEMENT_ONLY, testStandardXml(XML_ENUM_ELEMENT_ONLY));
+  }
+
+  /**
+   * The purpose of this test is to parse an XML element to an objects's member variable, whereas
+   * there are additional nested elements in the tag.
+   */
+  @Test
+  public void testParseToEnumElementTypeWithNestedElement() throws Exception {
+    assertEquals(XML_ENUM_ELEMENT_ONLY, testStandardXml(XML_ENUM_ELEMENT_ONLY_NESTED));
+  }
+
+  /**
+   * Private Method to handle standard parsing and mapping to {@link AnyTypeEnumElementOnly}
+   *
+   * @param xmlString XML String that needs to be mapped to {@link AnyTypeEnumElementOnly}
+   * @return Returns the serialized string of the XML Objects
+   * @throws Exception Thrown if there as an issuing when processing the XML.
```

Comment:
thrown if there is an issue processing the XML
no period

Reply:
fixed.